### ISSUE TITLE
Master not compatible with 9

### ISSUE THREAD

**Issue by mikestreety** [jpmschuler/TYPO3-tvplus-contentslide#7]
Hey, thanks for this extension - it is a lifesaver!

I noticed while trying to install the extension that the code on the `master` branch is not that of the code in the `v9` tag. Would it be possible to get `master` up-to-date please?

Many thanks!

**Comment by jpmschuler** [comment]
TL;DR: You didn't miss anything, just use v9.0.1

Yeah it is a lifesaver, however I just maintain its compatibility, the code is from EXT:kb_tv_cont_slide // Bernhard Kraft.

Sorry for the dirty repo. Master diverted somehow due to a local script error and unnecessary depandabot changes using the wrong branch. This is now all fixed. The master branch was deleted in the process, as it was not connected to the trunk anymore (which is and always was develop - just the Github Settings were still pointing to the wrong branch). 

Fell free to use `dev-develop` if you want the "trunk" edition of the code (what others would expect historically as a `dev-master`.
Nevertheless there are no relevant changes for 9LTS there. The current not released changes include

- softening/removing dependencies in order to allow easier testing for 10LTS and EXT:templavoila^8
- changing some text to adhere to the correct spelling of TemplaVoilà! (thus not worth releasing IMHO).

`dev-release` would point to the "stable" branch - however you could just require `*` than, too.

(Feel free to reopen or create a new issue if I misunderstood)

**Comment by mikestreety** [comment]
Thanks for the swift response! You didn't misunderstand at all. It had done a quick `git clone` into my ext folder then deleted the `.git` folder and couldn't work out why I had version 2 installed :)
